src/main/scala/com/fortysevendeg/translatebubble/ui/bubbleservice/Composer.scala

```diff
@@ -0,0 +1,346 @@
+package com.fortysevendeg.translatebubble.ui.bubbleservice
+
+import android.app.Service
+import android.content.Context
+import android.graphics.{PixelFormat, Point}
+import android.support.v4.view.ViewConfigurationCompat
+import android.view.ViewGroup.LayoutParams._
+import android.view.WindowManager.LayoutParams
+import android.view.WindowManager.LayoutParams._
+import android.view._
+import com.fortysevendeg.macroid.extras.DeviceMediaQueries._
+import com.fortysevendeg.translatebubble.R
+import com.fortysevendeg.translatebubble.modules.analytics.AnalyticsServicesComponent
+import com.fortysevendeg.translatebubble.modules.clipboard.ClipboardServicesComponent
+import com.fortysevendeg.translatebubble.modules.notifications.{NotificationsServicesComponent, ShowTextTranslatedRequest}
+import com.fortysevendeg.translatebubble.modules.persistent.PersistentServicesComponent
+import com.fortysevendeg.translatebubble.ui.commons.Strings._
+import com.fortysevendeg.translatebubble.ui.components.{ActionsView, BubbleView, ContentView}
+import com.fortysevendeg.translatebubble.utils.{Notification, Bubble}
+import macroid.FullDsl._
+import macroid.{Contexts, Ui}
+
+trait Composer {
+
+  self: Service
+    with Contexts[Service]
+    with AnalyticsServicesComponent
+    with PersistentServicesComponent
+    with ClipboardServicesComponent
+    with NotificationsServicesComponent =>
+
+  protected var widthScreen: Int = 0
+
+  protected var heightScreen: Int = 0
+
+  object BubbleStatus extends Enumeration {
+    type BubbleStatus = Value
+    val FLOATING, CONTENT = Value
+  }
```

Comment:
IMO this should be a `sealed case object`[1](http://underscore.io/blog/posts/2014/09/03/enumerations.html)